CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+## GitHub Flow
+
+For more information see [GitHub Flow](https://githubflow.github.io/)
+
+GitHub Flow is very lightweight (especially compared to GitFlow).
+This workflow uses only two kinds of branches:
+
+- Feature branch
+- Main branch (previously called master)
+
+The `feature` branches are used to develop new features as well as fixes.
+These branches are usually created out of main.
+
+Anything in the `main` branch is deployable.
+The `main` branch is expected to be deployed regularly and is considered stable.
+
+### How to work with GitHub Flow
+
+For more information see [GitHub Flow](https://githubflow.github.io/)
+
+TL;DR
+
+1. anything in the main branch is deployable
+2. create descriptive branches off of main
+3. push to named branches constantly
+4. open a pull request at any time
+5. merge only after pull request review
+6. deploy immediately after review
+
+## Working with Forks and Pull Requests
```

Comment:
A Pull Request template could be useful.

Reply:
Indeed. I made a ticket in the backlog.